operators/cc-operator/0.7.0/manifests/cc-operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,474 @@
+                ports:
+                - containerPort: 8443
+                  name: https
+                  protocol: TCP
+                resources: {}
+              - args:
+                - --health-probe-bind-address=:8081
+                - --metrics-bind-address=127.0.0.1:8080
+                - --leader-elect
+                command:
+                - /manager
+                env:
+                - name: CCRUNTIME_NAMESPACE
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.namespace
+                image: quay.io/confidential-containers/operator:v0.7.0
+                livenessProbe:
+                  httpGet:
+                    path: /healthz
+                    port: 8081
+                  initialDelaySeconds: 15
+                  periodSeconds: 20
+                name: manager
+                readinessProbe:
+                  httpGet:
+                    path: /readyz
+                    port: 8081
+                  initialDelaySeconds: 5
+                  periodSeconds: 10
+                resources:
+                  limits:
+                    cpu: 200m
+                    memory: 100Mi
+                  requests:
+                    cpu: 100m
+                    memory: 20Mi
+                securityContext:
+                  allowPrivilegeEscalation: false
+              securityContext:
+                runAsNonRoot: true
+              serviceAccountName: cc-operator-controller-manager
+              terminationGracePeriodSeconds: 10
+      permissions:
+      - rules:
+        - apiGroups:
+          - ""
+          resources:
+          - configmaps
+          verbs:
+          - get
+          - list
+          - watch
+          - create
+          - update
+          - patch
+          - delete
+        - apiGroups:
+          - coordination.k8s.io
+          resources:
+          - leases
+          verbs:
+          - get
+          - list
+          - watch
+          - create
+          - update
+          - patch
+          - delete
+        - apiGroups:
+          - ""
+          resources:
+          - events
+          verbs:
+          - create
+          - patch
+        serviceAccountName: cc-operator-controller-manager
+    strategy: deployment
+  installModes:
+  - supported: false
+    type: OwnNamespace
+  - supported: false
+    type: SingleNamespace
+  - supported: false
+    type: MultiNamespace
+  - supported: true
+    type: AllNamespaces
+  keywords:
+  - confidential containers
+  - ccruntime
+  - kata
+  - vm tee
+  links:
+  - name: Confidential Containers
+    url: https://github.com/confidential-containers
+  maintainers:
+  - email: fidenco
+    name: Fabiano
+  - email: jensfr
+    name: Jens
+  - email: bpradipt
+    name: Pradipta
+  maturity: alpha
+  provider:
+    name: Confidential Containers Community
+    url: https://github.com/confidential-containers
+  version: 0.0.1
```

Comment:
this needs to be 0.7.0 and you need to add
`  replaces: cc-operator.v0.5.0`  (same indentation as 'version')

maintainers should be 
`  maintainers:
  - email: cncf-ccontainers-maintainers@lists.cncf.io
    name: Jens Freimann
  - email: cncf-ccontainers-maintainers@lists.cncf.io
    name: Pradipta Banerjee
  - email: cncf-ccontainers-maintainers@lists.cncf.io
    name: Fabiano Fidencio
`

Reply:
Done